src/platform/Linux/Logging.cpp

```diff
@@ -25,7 +29,14 @@ namespace Platform {
  */
 void LogV(const char * module, uint8_t category, const char * msg, va_list v)
 {
-    printf("CHIP:%s: ", module);
+    struct timeval tv;
+
+    // Should not fail per man page of gettimeofday(), but failed to get time is not a fatal error in log. The bad time value will
+    // indicate the error occurred during getting time.
+    gettimeofday(&tv, nullptr);
+
+    printf("[%" PRIu64 ".%06" PRIu64 "][%ld] CHIP:%s: ", static_cast<uint64_t>(tv.tv_sec), static_cast<uint64_t>(tv.tv_usec),
```

Comment:
Could we create some define to make this feature configurable? Depends on the logging output, the logs may be timestamped by `syslog` itself, right?

Reply:
When putting logs to syslog, we may use log redirect  Thus this function won't be called.